drivers/spi/spi_ll_stm32.c

```diff
@@ -718,6 +830,11 @@ static int transceive(const struct device *dev,
 	spi_stm32_cs_control(dev, true);
 
 #ifdef CONFIG_SPI_STM32_INTERRUPT
+
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi)
+	LL_SPI_EnableIT_EOT(spi);
```

Comment:
Is it safe operation to perform when fifo is not used ?
Why not doing it under fifo-enabled condition ?

Reply:
My bad, that should happen only if FIFO is enabled.